sphinx/ext/mathjax.py

```diff
@@ -79,7 +79,7 @@ def install_mathjax(app: Sphinx, pagename: str, templatename: str, context: Dict
                              'mathjax extension to work')
 
     domain = cast(MathDomain, app.env.get_domain('math'))
-    if domain.has_equations(pagename):
+    if app.registry.html_assets_policy == 'always' or domain.has_equations(pagename):
```

Comment:
@tk0miya I don't like too much how this line is checking for the `always` policy. Do you have a better suggestion for this?